src/assets/css/global/_main.scss

```diff
@@ -665,3 +670,32 @@ km-cni-version {
 .km-option-divider.mat-divider {
   margin: 10px 0;
 }
+
+km-machine-deployment-list {
```

Comment:
Please use class instead of component selector. (same comment for other similar changes in this PR)